Maths/test/Fibonacci.test.ts

```diff
@@ -0,0 +1,27 @@
+import { fibonacciValue } from "../Fibonacci";
+
+describe("FibonacciValue", () => {
+  it("Should return correct value", () => {
```

Comment:
redundant test names; just move everything into a `test` block if there is nothing to group your tests by